loader/include/Geode/Enums.hpp

```diff
@@ -257,8 +257,17 @@ enum class StatKey {};
 enum class TextStyleType {};
 enum class InputValueType {};
 enum class GJInputStyle {};
-enum class GJDifficultyName {};
-enum class GJFeatureState {};
+enum class GJDifficultyName {
```

Comment:
i think this is wrong, logging it to the console in GJDifficultySprite::create results in 0 and 1 appearing on both normal and demon difficulties

Reply:
No it is not.

Reply:
non-demon:
![image](https://github.com/geode-sdk/geode/assets/37266659/f4fe61e4-8a87-48bb-b034-29f3c2b6f1e0)

demon:
![image](https://github.com/geode-sdk/geode/assets/37266659/c5fcbdbe-a9a6-4efb-a47b-79d9716aeda2)

it does appear that 0 is passed to GJDifficultySprite::create when in LevelBrowserLayer/LevelListLayer, but in LevelInfoLayer 1 is used. While GJDifficultySprite::updateDifficultyFrame is always passed 1?

Reply:
Let me explain my stance:
If `GJDifficultyName` is 1, the demon difficulty will be shown in text if it is a demon, otherwise, it will revert to default.
It is condensed if `GJDifficultyName` is 0.
`updateDificultyFrame` is only used in `LevelInfoLayer`.

Reply:
yeah so my last point still stands, a more appropriate name would be DefaultText and ShortText imo